src/gc/impl/precise/gc.d

```diff
@@ -1191,7 +1353,15 @@ class ConservativeGC : GC
     }
 
 
+<<<<<<< HEAD:src/gc/impl/precise/gc.d
```

Comment:
Artifacts of a merge conflict? This might contribute to your rebase issues.
BTW: GCStats was moved to core.memory.GC.Stats.